lib/modules/datasource/rubygems/versions-endpoint-cache.ts

```diff
@@ -140,28 +142,26 @@ export class VersionsEndpointCache {
    * At any given time, there should only be one request for a given registryUrl.
    */
   private async getCache(registryUrl: string): Promise<VersionsEndpointResult> {
-    const cacheKey = `rubygems-versions-cache:${registryUrl}`;
+    const oldResult = memCache.get(registryUrl);
```

Comment:
why changing the cache key?

Reply:
It isn't going to the long-term cache, this structure can only have one element. In further PRs, I'll make it a variable instead of `Set`, but for now it's enough to simplify this `cacheKey` thing. Otherwise, I would need to add `getCacheKey(registryUrl)` method instead, as it's being used in several places.